apps/backend/src/modules/project-groups/project-group.entity.ts

```diff
@@ -0,0 +1,99 @@
+import { type Entity } from "~/libs/types/types.js";
+
+import { type ProjectGroupCreateResponseDto } from "./libs/types/types.js";
+
+class ProjectGroupEntity implements Entity {
+	private id: null | number;
+	private name!: string;
+	private permissionIds!: { id: number }[];
+	private projectId!: { id: number };
+	private userIds!: { id: number }[];
```

Comment:
You are using the similar approach as I was, I refactored it a little, maybe take a look at this [comment](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/56#discussion_r1735003814)

Reply:
I'll take a look, thanks